network/xil_eth/xil_eth.c

```diff
@@ -0,0 +1,654 @@
+/***************************************************************************//**
+ * @file xil_eth.c
+ * @brief Implementation of ethernet functions for network.
+ * @author Mihail Chindris (mihail.chindrs@analog.com)
```

Comment:
some typos in comments:
- your name specifically
- LINUX_SOCKET_H_ i guess is a remnant from a template